DQMOffline/Configuration/python/DQMOffline_Certification_cff.py

```diff
@@ -45,5 +45,9 @@
     sipixelCertification # segfaults when included
```

Comment:
@mmusich do you know/recal by chance why these lines are here? They actually try to remove Pixel certification on any Phase1 wf as I am now trying to do. Thanks

Reply:
`git blame` points to https://github.com/cms-sw/cmssw/pull/17855

Reply:
Thanks @makortel !
I am afraid the removal was not fully effective, and now I am actually removing the PixelCertification in Phase1 wf, hence the change.
However, if I don't remove it I get:
 Calling method for module SiPixelCertification/'sipixelCertification'
Exception Message:
A std::exception was thrown.
DQM object not found: Pixel/EventInfo/reportSummary